spec/controllers/admin/membership_submissions_controller_spec.rb

```diff
@@ -9,7 +9,89 @@
     end
   end
 
-  pending "GET #pending"
-  pending "POST #pending"
-  pending "POST #reject"
+  describe "GET #pending" do
+    describe "when pending submissions are zero" do
+      before do
+        allow(@controller).to receive(:require_admin!).and_return(true)
```

Comment:
Not a big deal, but it would be great to move this into a helper method in spec/support/*** so we could abstract it a bit more.  Something like `authorize_admin!` and a corresponding method to perhaps deny authorization so we can test that too.  I think it would make it easier to read these controller specs.